README.rst

```diff
@@ -36,6 +36,44 @@ administrator must populate initially.
 #. Log in to the headnode, set up a PXE server, reboot the nodes, and deploy an
    operating system on them via the network.
 
+Requirements
+============
+
+Required software/hardware for running a production HaaS include:
+
+* Network switches:
+
+  * At least one switch from the Cisco Nexus 5xxx or Dell PowerConnect 55xx families
+  * For environments including more than one switch, all VLANs must be trunked to all managed switches
+
+* A single node that has the following:
+
+  * A webserver capable of supporting the WSGI standard (Apache/mod_wsgi is the only one tested)
+  * python 2.7, with the ability to install packages via pip
+  * Access to:
+
+    * The Internet or intranet (a way for users to connect to the HaaS service)
+    * The administrative telnet IP on the managed switches
+
+  * Currently only CentOS and RHEL 7.x have been tested, though any node that otherwise meets these requirements should function.
+
+* Database: a Postgres database server. Sqlite works but is not recommended for production.
+
+Optional components
```

Comment:
I guess I see these as optional, but the current API doesn't necessarily reflect that. Would you prefer merging in the current optional section?

Re: IPMI, in certain trusted environments, users may have direct access to IPMI anyhow and not require HaaS. Perhaps to make the point, with the new OBM drivers we could include a "null" driver?

Re: Headnodes, until we've completed separating this functionality into its own service we could add a headnodes=False config option that would cause the calls to throw an exception.